index.js

```diff
@@ -761,14 +762,15 @@ const runTests = async () => {
 
     debug(`Cypress tests: ${testResults.totalFailed} failed`)
 
-    if (dashboardUrl) {
-      debug(`Dashboard url ${dashboardUrl}`)
+    if (resultsUrl) {
+      debug(`resultsUrl ${resultsUrl}`)
     } else {
-      debug('There is no Dashboard url')
+      debug('There is no resultsUrl')
     }
 
     // we still set the output explicitly
-    core.setOutput('dashboardUrl', dashboardUrl)
+    core.setOutput('dashboardUrl', dashboardUrl) // deprecated and retained for backward compatibility
```

Comment:
```suggestion
    core.setOutput('dashboardUrl', resultsUrl) // deprecated and retained for backward compatibility
```

And then declaring the const above is no longer necessary.

Reply:
You're right. This could be simplified.

Reply:
@chrisbreiding 
Many thanks for your review and suggestions! 👍🏻 
I have made the changes.